appveyor.yml

```diff
@@ -1,37 +1,41 @@
 # Based on the "trust" template v0.1.1
 # https://github.com/japaric/trust/tree/v0.1.1
 
+os: Visual Studio 2017
+
 environment:
   global:
-    RUST_VERSION: stable
     CRATE_NAME: cobalt
     APPVEYOR_CACHE_SKIP_RESTORE: true
+    MSBUILD_NODE_CONNECTION_TIMEOUT: 0
 
   matrix:
     # Stable channel
-    - TARGET: x86_64-pc-windows-gnu
-      CHANNEL: stable
-    - TARGET: i686-pc-windows-msvc
-      CHANNEL: stable
     - TARGET: x86_64-pc-windows-msvc
       CHANNEL: stable
+      TOOLCHAIN: msvc
+      PLATFORM: x86_64
     # Beta channel
     - TARGET: x86_64-pc-windows-msvc
       CHANNEL: beta
+      TOOLCHAIN: msvc
+      PLATFORM: x86_64
```

Comment:
What is `PLATFORM`?

Should we set it globally?

Reply:
Good catch, it was used in the `install` part to call the correct config `.bat` file according to i686 vs x86_64. To be deleted :)